addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -307,11 +309,10 @@ QUnit.module('web_editor', {}, function () {
                 const openingProm = new Promise(resolve => {
                     $colorpicker.one('shown.bs.dropdown', () => resolve());
                 });
-                await testUtils.dom.click($colorpicker.find('button:first'));
+                await testUtils.dom.click($colorpicker.find('> :first'));
                 return openingProm;
             }
 
-            await new Promise((resolve)=>setTimeout(resolve, 50));
```

Comment:
I tried to minimize the amount of timeout, the one I put makes it work for me. I think the issue is with the testutils thing.

Reply:
I think you should not midify this line because:
- we usually go for minimum diff, especially in stable
- the fact that 10ms works for you doesn't mean it will work on runbot, and even if it does, it might sometimes be enough and sometimes not due to some race conditions or the runbot being slower sometimes due to cpu charge. If this test is sometimes red because of the reduced wait time, it is a big issue because random people will have their branch red for no reason. It could even make staging build fail, thus blocking multiple PRs at the same time etc. It's a very big deal, in fact. Usually we avoid introducing those random `setTimeout` because of that, as it is impossible to really know when the delay is safe. This one here is probably old.
- the gain is ultra small (only 40ms) and therefore not worth going against the above two points